js/Landing.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+   color: white;
+`;
+
+const LandingNavbar = styled.div`
+  h2{
+    float:left;
+    margin:2px;
+    font-weight: normal;
+  }
+  a{
+    float:right;
+    margin:5px;
+    font-weight: bolder;
+  }
+  font-family: Roboto
```

Comment:
It'd be good to include some backup font families for if Roboto is not available on the user's computer. 
Generally, this can be done by simply adding a comma after each font family.
The browser will go through the list until it is able to apply one of them e.g.
`font-family: "Bungee Shade", Helvetica, Ariel, sans-serif;`

It is also best practice to use so-called ["web-safe fonts"](https://websitesetup.org/web-safe-fonts-html-css/)

Reply:
Thanks. done in src/js/main.css